design-proposals/dedicated-migration-network.md

```diff
@@ -0,0 +1,94 @@
+# Overview
+This design provides a way to migrate VMIs (Virtual Machine Instances) over a dedicated network.
+VMI migrations currently happen over the main Kubernetes network all pods are connected to.
+
+## Motivation
+In order for VMI migrations to succeed, there must be a fair amount of bandwidth available.
+If for example the memory of a migrating VMI changes faster than the current speed of the main network, the migration will never succeed.
+This is especially problematic during mass eviction events like platform upgrades.
+This problem is currently mitigated by throttling the number of concurrent migrations and using long timeouts.
+
+## Goals
+- Allow a cluster admin to setup and dedicate a secondary physical cluster network to migrations.
+- Keep the main network as the default medium for migrations.
+
+## Non Goals
```

Comment:
Per my understanding, this proposal does not cover QoS and the new network prioritization over the other networks.
I mention this, because this point is sometime essential for prioritizing the migration traffic over others.

Can we mention this as well in the non-goals?